go.mod

```diff
@@ -3,26 +3,26 @@ module github.com/deislabs/ratify
 go 1.16
 
 require (
-	github.com/Azure/azure-sdk-for-go v61.3.0+incompatible
+	github.com/Azure/azure-sdk-for-go v63.4.0+incompatible
 	github.com/AzureAD/microsoft-authentication-library-for-go v0.4.0
-	github.com/docker/cli v20.10.12+incompatible
-	github.com/docker/distribution v2.8.0+incompatible
+	github.com/docker/cli v20.10.14+incompatible
+	github.com/docker/distribution v2.8.1+incompatible
 	github.com/google/go-containerregistry v0.8.1-0.20220125170349-50dfc2733d10
 	github.com/gorilla/mux v1.8.0
 	github.com/notaryproject/notation-go-lib v1.0.0-alpha-1
 	github.com/open-policy-agent/frameworks/constraint v0.0.0-20211029184625-8b4a99a9a65a
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.2
-	github.com/oras-project/artifacts-spec v1.0.0-draft.1.1
+	github.com/oras-project/artifacts-spec v1.0.0-rc.1
```

Comment:
upgrading to rc.1 is tracked in #168 . The descriptor fields haven't really changed apart from the mediatype being required now so I think it should work. The plan was to hold off on bumping the clients (including ratify) to rc.1 until distribution had rc1 support. I'm fine either way but just wanted to point this out. I think it should be fine

Reply:
ah thanks for point this out. I ll revert this, dependabot might open a PR , but we ll just leave it open.